docs/01_nodeos/07_concepts/15_private_chain_access/index.md

```diff
@@ -0,0 +1,79 @@
+
+## Concept
+
+Most businesses need to keep data private for regulatory compliance, customer privacy, or both. Since blockchain solutions have a history of public data and open architecture, businesses need certain privacy and security guarantees before they can adopt blockchain technology. The *Private Access Control* feature aims to bring EOSIO closer to the Enterprise by providing a general business security solution that can be used across a variety of [Use Cases](#use-cases).
+
+[[info | Access controls via consensus]]
+| Along with the inherent features of the EOSIO blockchain, the *Private Access Control* feature achieves security, robustness, and coordination through EOSIO consensus. The protocol feature can be enabled and participants added or removed from a *security group* only after the block that contains the corresponding transaction is approved by a supermajority of producer participants and the block becomes irreversible.
+
+## Requirements
+
+The *Private Access Control* feature must meet the following requirements:
+- The SECURITY_GROUP protocol feature must be activated first via EOSIO consensus.
+- Once activated, only nodes with valid signed credentials are allowed to connect.
+- valid nodes must establish TLS connections to communicate securely with other peers.
+- Once connected, all peer-to-peer communications must be encrypted - TLS satisfies this.
+- Optionally, a *security group* can be used to manage participant data access to the network:
+  * If at least one participant is added to the group, data access control is enabled.
+  * If no participants are added to the group, data access control is disabled.
+  * When data access control is disabled, all participants in the group are allowed data access.
+  * When data access control is enabled, only those participants in the group are allowed data access.
+
+The secure connection requirements are met by establishing TLS connections among the participant nodes. This is considered sufficient from a security standpoint since the peer-to-peer communications are encrypted. The optional data access control requirements are met through the *security group* which adds further data access control on a per-participant basis. Adding and removing participans within the *security group* as well as additional housekeeping operations are implemented through EOSIO WASM host functions, which can be invoked from wrapper actions within custom smart contracts.
```

Comment:
Tiny typo: 
"Adding and removing participans" -> participants

Reply:
Oh man... :)